oppia_android_application.bzl

```diff
@@ -390,30 +394,30 @@ def oppia_android_application(name, config_file, proguard_generate_mapping, **kw
             tags = ["manual"],
         )
 
-def declare_deployable_application(name, aab_target):
+def generate_universal_apk(name, aab_target):
     """
-    Creates a new target that can be run with 'bazel run' to install an AAB file.
+    Creates a new installable universal APK target for the provided AAB target.
+
+    This new target is installable via 'adb install'. This should generally never be used for direct
+    development installs as it will be far less performant than 'bazel mobile-install'.
```

Comment:
I find this ambiguous, it is not clear to me whether the warning is against using 'adb install' or `bazel mobile-install //:oppia_prod_universal_apk`

Reply:
Ah, yeah this is just very confusing. I decided to remove the 'adb install' bit since it's not important to know, and tried to make the other parts of the documentation here a bit clearer. I did add a new 'adb install' section to cover a detail I didn't think about originally--there are cases when mobile-install can actually be detrimental to use. It's mainly useful for development since it can substantially decrease install time, but it runs different code than a real production APK so it's less good for testing production builds.

Does this look better to you?

Reply:
This is clear now.